config.json

```diff
@@ -618,7 +618,38 @@
       "name": "Type Conversion"
     }
   ],
-  "key_features": [],
+  "key_features": [
+    {
+      icon: "web",
```

Comment:
I would suggest the first listed feature to be that Dart is a general-purpose language that allows it to be flexible and used in a variety of use-cases.

Reply:
The syntax for JSON is for keys to be treated as strings. This would need to be resolved.